pkg/services/secrets/fakes/fake_store.go

```diff
@@ -3,8 +3,9 @@ package fakes
 import (
 	"context"
 
+	"github.com/grafana/grafana/pkg/services/sqlstore"
```

Comment:
This will result into cyclic dependency in tests because we use FakeSecretsStore in sqlstore package's tests.

Reply:
```
github.com/grafana/grafana/pkg/services/sqlstore
package github.com/grafana/grafana/pkg/services/sqlstore
        imports github.com/grafana/grafana/pkg/services/secrets/fakes: import cycle not allowed in test
FAIL    github.com/grafana/grafana/pkg/services/sqlstore [setup failed]

```